dashboard/src/pages/Dashboard/Dataset/DatasetStart.tsx

```diff
@@ -77,6 +80,54 @@ export const DatasetStart = () => {
     },
   );
 
+  const reloadChunkCount = () => {
+    const datasetId = currDatasetId();
+    if (!datasetId) {
+      console.error("Dataset ID is undefined.");
+      return;
+    }
+
+    void (async () => {
+      try {
+        const response = await fetch(`${api_host}/dataset/usage/${datasetId}`, {
+          method: "GET",
+          headers: {
+            "TR-Dataset": datasetId,
+            "Content-Type": "application/json",
+          },
+          credentials: "include",
+        });
+
+        if (!response.ok) {
+          throw new Error("Failed to fetch dataset usage");
+        }
+
+        const newData = (await response.json()) as DatasetUsageCount;
+        if (usage()?.chunk_count === newData.chunk_count) {
+          createToast({
+            title: "No Update Needed",
+            type: "info",
+            message: "Chunk count is already up to date",
+          });
+        } else {
+          mutateUsage(() => newData);
+          createToast({
+            title: "Updated",
+            type: "success",
+            message: "Updated chunk count",
```

Comment:
```suggestion
            message: `Successfully updated chunk count: ${new_count-prev_count} chunks have been ${new_count-prev_count > 0 ? added : removed} since last update.`
```